dev/documentation/en/utilities-translation.md

```diff
@@ -48,6 +47,17 @@ msgstr "Completed: {$dateCompleted}"
 $label = __('common.completed.date', ['dateCompleted' => $dateCompleted]);
 ```
 
+Use the `__p()` function to work with pluralized locale keys.
+
+```php
+// 1 = one submission
+$submissions = __p('common.submissionsCount', 1, ['param' => 1]);
+// 10 = zero or more than one submission
+$submissions = __p('common.submissionsCount', 10, ['param' => 10]);
```

Comment:
I'm not sure I understand this example. Can we use an example with a plural word, like:

> You have {$param} message[s] pending.

Also, what's the third plural form? Is it dual or what? I know in Arabic plural forms are a little bit funny:

- 1 = singular
- 2 = dual
- 3-10 = plural
- 11-* = singular